components/dashboard/preview/AccountSwitcher.tsx

```diff
@@ -1,24 +1,28 @@
 import * as React from 'react';
+import { PopoverAnchor } from '@radix-ui/react-popover';
 import { useCommandState } from 'carloslfu-cmdk-internal';
 import { cx } from 'class-variance-authority';
 import clsx from 'clsx';
 import { flatten, groupBy, uniqBy } from 'lodash';
 import { Check, ChevronsUpDown, PlusCircle, Star } from 'lucide-react';
 import memoizeOne from 'memoize-one';
+// eslint-disable-next-line no-restricted-imports -- components/Link does not currently accept a ref, whichis required when used 'asChild' of Button
```

Comment:
Is that still true, since the direct child of the popover is a div?

Reply:
@Betree the issue is not for the popover but for the Link inside a Button here:

https://github.com/opencollective/opencollective-frontend/pull/9626/files#diff-4f44c2117b532f4e6afae6c4436d6b14a3e86503d022afeb55cc80c0d9660a26R329-R346